UnitsNet/CustomCode/Quantities/ReciprocalLength.extra.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using UnitsNet.Units;
+
+namespace UnitsNet
+{
+    public partial struct ReciprocalLength
+    {
+        /// <summary>
+        /// Calculates the inverse or <see cref="Length"/> of this unit.
```

Comment:
Nitpick, but `ReciprocalLength` is a _quantity_ and not a _unit_. And since we don't map from InverseYard to Yard, maybe we could describe it like this:

`Returns the inverse quantity <see cref="Length"/> with the unit <see cref="LengthUnit.Meter />.`

Reply:
Updated.